docs/codeql/writing-codeql-queries/about-codeql-queries.rst

```diff
@@ -78,6 +77,7 @@ When writing your own alert queries, you would typically import the standard lib
 - Java: ``java``
 - JavaScript/TypeScript: ``javascript``
 - Python: ``python``
+- Ruby: ``ruby``
```

Comment:
I notice that most of the Ruby queries that I've seen use something more than just `ruby`. For example: https://github.com/github/codeql/blob/main/ruby/ql/src/queries/security/cwe-089/SqlInjection.ql.

Is this expected? It changed recently in: https://github.com/github/codeql/pull/10376.

Reply:
Good spot. This is indeed expected. There are plans that more languages might make similar changes to their libraries, so i think linking to the 'CodeQL library guides' might help with any future changes here.